### PR TITLE
feat(schemas): create sso_connectors table

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Create the `sso_connectors` table.
Please check the inline comments for detailed field explanations. 

## Testing

N/A

## Checklist

- ~[ ] `.changeset`~
- ~[ ] unit tests~
- ~[ ] integration tests~
- ~[ ] necessary TSDoc comments~

### CODE REVIEW

File: packages/schemas/src/foundations/jsonb-types/sso-connector.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { z } from 'zod';
+
+export const ssoDomainsGuard = z.array(z.string());
+
+export type SsoDomains = z.infer<typeof ssoDomainsGuard>;
+
+export const ssoBrandingGuard = z.object({
+  logto: z.string().optional(),
```
Comment (darcyYe) [inline packages/schemas/src/foundations/jsonb-types/sso-connector.ts:8]: logo

File: packages/schemas/tables/sso_connectors.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
+create table sso_connectors (
+  tenant_id varchar(21) not null 
+    references tenants (id) on update cascade on delete cascade,
+  /** The globally unique identifier of the SSO connector. */
+  id varchar(128) not null,
+  /** The connector factory name of the SSO provider. */
+  provider_name varchar(128) not null,
+  /** The name of the SSO provider for display. */
+  connector_name varchar(128) not null,
```
Comment (darcyYe) [inline packages/schemas/tables/sso_connectors.sql:9]: nit: can simply use `name` or `displayName`

**Review by darcyYe — APPROVED** [review]

File: packages/schemas/src/foundations/jsonb-types/sso-connector.ts (hunk 1 above)
Comment (simeng-li) [reply packages/schemas/src/foundations/jsonb-types/sso-connector.ts]: updated

File: packages/schemas/tables/sso_connectors.sql (hunk 2 above)
Comment (simeng-li) [reply packages/schemas/tables/sso_connectors.sql:9]: aligned with the name on design

**Review by wangsijie — APPROVED** [review]

**Pull request merged**